src/core/server/integration_tests/saved_objects/migrations/group1/jest.integration.config.js

```diff
@@ -7,6 +7,7 @@
  */
 
 module.exports = {
+  bail: true, // only report 1 issue
```

Comment:
Yeah, I'm not sure about this.

Yes, when the test suite fails during `setup`, it avoid our GH notifications being flooded. 

BUT it also means that in case of real failures (e.g when developing on the domain), we can't rely on the CI to list all the failures and we must run the suite(s) manually locally. 

So... yeah, idk

Reply:
yea, I was testing different ways to resolve the failures.